firmware/src/widgets/stockwidget/StockWidget.cpp

```diff
@@ -131,7 +134,11 @@ void StockWidget::displayStock(int8_t displayIndex, StockDataModel &stock, uint3
         m_manager.fillTriangle(110 + arrowOffsetX, 132 + arrowOffsetY, 130 + arrowOffsetX, 132 + arrowOffsetY, 120 + arrowOffsetX, 120 + arrowOffsetY, TFT_GREEN);
         m_manager.drawArc(centre, centre, 120, 118, 0, 360, TFT_GREEN, TFT_GREEN);
     }
-    m_manager.drawString(stock.getPercentChange(2) + "%", centre, 48, bigFontSize, Align::MiddleCenter);
+    if (!m_stockchangeformat) {
+        m_manager.drawString(stock.getPercentChange(2) + "%", centre, 48, bigFontSize, Align::MiddleCenter);
+    } else {
+        m_manager.drawString(stock.getPriceChange(2) + stock.getCurrencySymbol(), centre, 48, bigFontSize, Align::MiddleCenter);
```

Comment:
The currency symbol for the High/Low price is **before** the value, and here it's **after**.  Let's be consistent (even if some locale would do it differently, let's not open that can of worms).

Likewise. the hardcoded "2" (decimals) is inconsistent with the high/low price (which looks like it uses the data straight from the API). Should we just do the same?

Disclaimer: code not tested.